docs/getting-started/running.md

```diff
@@ -32,4 +32,11 @@ There are two ways of running the `deploy` command:
    ![Deploy Run Configuration](../img/deploy_config.png)
 
 3. Click on the run button in the top right corner of the IntelliJ window to deploy your robot
-   program to the robot.
\ No newline at end of file
+   program to the robot.
+
+## Simulating Robot Code
+
+FRC code made with kotlin can be simulated on your computer, just like robot code written with Java, Python, or C++. To do so,either follow the "Deploying via IntelliJ" or "Deploying manually via the command line" instructions above, but replace the "deploy" keyword with the "simulateJava" keyword(I.E `./gradlew simulateJava` or `./gradlew.bat simulateJava` on windows).
+
+WPILib's simulation allows you to run your code locally on your computer(without an actual robot). This should be used in junction with the WPILib simulation classes to simulate your mechanisms; read more about the physics simulation API [here](
```

Comment:
Maybe also discuss hardware sim

Reply:
I feel like hardware sim is already explained pretty well in the wpilib docs(I don't think kotlin does anything better for hardware sim than java does).

Reply:
Personally i think that wpilib should endorse teams using abstract classes/io implementations/vendor-specific sim support over EncoderSim and such because those sim classes are only available for a limited amount of wpilib classes and aren't that great of an API to begin with